### PR TITLE
Replace Loading Icon

### PR DESCRIPTION
**Pull request by thomasdax98**
Replaces `CircularProgress` with our `BallTriangle` icon in the `Loading` component

Unfortunately, that's a breaking change because the `CircularProgressProps` was passed through. I replaced that with the `SvgIconProps`, although it's debatable if passing the props through is needed at all.

### CODE REVIEW

**Review by jamesricky — CHANGES_REQUESTED** [review]
Please add a changeset for the incompatible change.

File: packages/admin/cms-admin/src/common/header/UserHeaderItem.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export function UserHeaderItem(props: UserHeaderItemProps): React.ReactElement {
     if (loading || !data)
         return (
             <LoadingWrapper>
-                <Loading behavior="fillParent" size={20} color="inherit" />
+                <Loading behavior="fillParent" color="inherit" />
```
Comment (thomasdax98) [inline packages/admin/cms-admin/src/common/header/UserHeaderItem.tsx:63]: TODO: size

Comment (thomasdax98) [reply packages/admin/cms-admin/src/common/header/UserHeaderItem.tsx:63]: @jamesricky `size` seems to be useful. It seems bothersome to always create a styled component to adjust the size. 
Should we (re)implement the size prop for `Loading`? What do you think?

**Review by jamesricky — DISMISSED** [review]

File: .changeset/smart-kangaroos-marry.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": major
+---
+
+Change the icon used in the `Loading` component from Mui's `CircularProgress` to our `BallTriangle`
```
Comment (johnnyomair) [inline .changeset/smart-kangaroos-marry.md:5]:
```suggestion
Change the icon used in the `Loading` component from MUI's `CircularProgress` to our `BallTriangle`
```

File: packages/admin/admin/src/common/Loading.tsx [hunk 3]
```diff
@@ -58,3 +60,10 @@ const LoadingContainer = styled("div")<Required<Pick<LoadingProps, "behavior">>>
             justify-content: center;
         `}
 `;
+
+const StyledBallTriangle = styled(BallTriangle, {
+    shouldForwardProp: (propName) => propName !== "size",
+})<{ size: number }>`
+    width: ${({ size }) => `${size}px`};
+    height: ${({ size }) => `${size}px`};
+`;
```
Comment (johnnyomair) [inline packages/admin/admin/src/common/Loading.tsx:69]: BallTriangle is a SvgIcon component. I think we should use the recommended way for changing the size of the icon: https://mui.com/material-ui/icons/#size (i.e. remove the `size` prop and styled wrapper and use `fontSize` accordingly).

Comment (thomasdax98) [reply packages/admin/admin/src/common/Loading.tsx]: Okay, should be alright now

**Review by johnnyomair — APPROVED** [review]

**Review by jamesricky — APPROVED** [review]

**Pull request merged**